.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+## How to contribute to Open Bastion
+
+#### **Did you find a bug?**
+
+* You can either just create an issue on the GitHub repository or create a pull
```

Comment:
I would write it in this order : 

Check first that a similar issue is not already opened.

If one exists and you can bring something new, please comment on the issue.

If not, you can either create a new one on the GitHub repository or create a pull
request with your fix proposal. We will be happy to review and integrate it if
possible.

All the issues and pull requests need to follow the [issue template](https://github.com/open-bastion/open-bastion/blob/master/.github/ISSUE_TEMPLATE/bug_report.md).

Reply:
All good for me except the `All the issues and pull requests need to follow the issue template.`. Don't you prefer: `As much as possible, issues should follow the issue template`?

Reply:
Looks good to me!